src/main/webapp/assets/css/global-custom.css

```diff
@@ -540,3 +540,25 @@ body{
 .section-logo img{
     opacity: 0.5;
 }
+
+/*our projects display section*/
+.project-img {
+    padding: 15px;
+    width: 200px;
+    height: auto;
+}
+
+.project-card{
```

Comment:
I think this is not formatted. Can you please check that?

Reply:
fixed